includes/managers/controls.php

```diff
@@ -1337,4 +1349,26 @@ private function promotion_select_control( $title, $id ): string {
 			</div>
 		</div>';
 	}
+
+	private function is_style_control( $control_data ): bool {
+		$frontend_available = $control_data['frontend_available'] ?? false;
+		if ( $frontend_available ) {
+			return false;
+		}
+
+		if ( ! empty( $control_data['prefix_class'] ) ) {
+			return false;
+		}
+
+		if ( ! empty( $control_data['selectors'] ) ) {
+			return true;
+		}
+
+		$render_type = $control_data['render_type'] ?? '';
+		if ( 'ui' === $render_type ) {
+			return true;
+		}
+
+		return false;
+	}
```

Comment:
isn't it enough to just check for `selectors`?

Reply:
@StyleShit What did you do with controls with `selectors` and `prefix_classes` / `frontend_available`?